estimation/basecamp/moments_calculation.py

```diff
@@ -46,3 +48,455 @@ def get_moments(data):
             moments["Choice_Probability"][period].append(stat)
 
     return moments
+
+
+def transitions_out_to_in(data_subset):
+    counts_list = []
+    for period in np.arange(1, 39):
```

Comment:
` 39` is hard-coded over and over, please make endogenous.

Reply:
Done